app/components/nextflow_component.en.yml

```diff
@@ -0,0 +1,4 @@
+en:
+  name:
+    label: Name (Optional)
+    helper: A custom name will make it easier to search for this workflow execution in the future.
```

Comment:
**nitpick (blocking):** Can you change the helper text to `A custom name will it easier to search for this in the future.`

That way this also makes sense on the automated workflow executions page.

Reply:
Updated in b01e147